ygen/schema_tests/schema_test.go

```diff
@@ -80,3 +80,67 @@ func TestMultiKeyListRename(t *testing.T) {
 		t.Errorf("did not have correct name in newBGP, got: %v, want: 36040", *ni.Protocol[newBGP].Name)
 	}
 }
+
+func TestSimpleKeyAppend(t *testing.T) {
+	in := &exampleoc.Device{}
+	ni := &exampleoc.NetworkInstance{
+		Name: ygot.String("DEFAULT"),
+	}
+	if err := in.AppendNetworkInstance(ni); err != nil {
+		t.Errorf("AppendNetworkInstance(%v): did not get expected error, got: %v, want: nil", ni, err)
+	}
+
+	if _, ok := in.NetworkInstance["DEFAULT"]; !ok {
+		t.Errorf("AppendNetworkInstance(%v): did not find element after append, got: %v, want: true", ni, ok)
+	}
+}
+
+func TestMultiKeyAppend(t *testing.T) {
+	in := &exampleoc.Device{}
+	_, err := in.NewNetworkInstance("DEFAULT")
```

Comment:
(nit) Can we minimize the scope of err here?

if _, err := in.NewNetworkInstance(); err != nil {...}

Reply:
ACK, done, thanks.